components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -0,0 +1,227 @@
+import { LitElement, css, html } from 'lit-element';
+import { LocalizeStaticMixin } from '@brightspace-ui/core/mixins/localize-static-mixin.js';
+
+import '@brightspace-ui-labs/multi-select/multi-select-list';
+import '@brightspace-ui-labs/multi-select/multi-select-list-item';
+import { bodyCompactStyles } from '@brightspace-ui/core/components/typography/styles.js';
+import { getComposedChildren } from '@brightspace-ui/core/helpers/dom';
+
+const DROPDOWN_NAME = 'D2L-FILTER-DROPDOWN';
+const DROPDOWN_CATEGORY_NAME = 'D2L-FILTER-DROPDOWN-CATEGORY';
+const DROPDOWN_OPTION_NAME = 'D2L-FILTER-DROPDOWN-OPTION';
+
+class D2lAppliedFilters extends LocalizeStaticMixin(LitElement) {
+	static get properties() {
+		return {
+			for: { type: String },
+			_entries: { type: Array },
+			_selectedEntries: { type: Array },
+			_target: { type: Object },
+		};
+	}
+
+	static get styles() {
+		return [bodyCompactStyles, css`
+			:host {
+				display: block;
+			}
+
+			d2l-labs-multi-select-list {
+				flex: 1;
+			}
+
+			.d2l-applied-filters-wrapper {
+				display: flex;
+			}
+
+			.d2l-applied-filters-applied-filters-label {
+				margin-right: 0.25rem;
```

Comment:
We probably need to handle the RTL case here and include https://github.com/BrightspaceUI/core/blob/master/mixins/rtl-mixin.js